addons/docs/package.json

```diff
@@ -59,6 +60,9 @@
     "ts-dedent": "^1.1.0"
   },
   "devDependencies": {
+    "@types/doctrine": "^0.0.3",
+    "@types/enzyme": "^3.10.3",
```

Comment:
Is this needed?

Reply:
Without those types I couldn't compile the app witht `yarn dev`